Microsoft.DotNet.Interactive/FormattedValue.cs

```diff
@@ -1,13 +1,15 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+
 namespace Microsoft.DotNet.Interactive
 {
     public class FormattedValue
     {
         public FormattedValue(string mimeType, object value)
         {
-            MimeType = mimeType;
+            MimeType = mimeType ?? throw new ArgumentNullException(nameof(mimeType));
```

Comment:
This should also check for empty and whitespace, which are also not valid mime types.